addons/sys_rack/stringtable.xml

```diff
@@ -166,6 +169,7 @@
             <Chinesesimp>仪表板上升</Chinesesimp>
             <Spanish>P.Sup</Spanish>
             <French>T.Sup</French>
+            <German>Oberster Steckplatz</German>
```

Comment:
have you checked that it's not too long (decluttering of the Menu is vital)

Reply:
This one should be maximum 5 characters or a warning will be logged.